File: Lista de exercicio/projetoJAVA/src/projetojava/questao2.java
package projetojava;

class Carro1 {
    String marca;
    String modelo;
    int ano;
    int velocidade;

    Carro1(String marca, String modelo, int ano) {
        this.marca = marca;
        this.modelo = modelo;
        this.ano = ano;
        this.velocidade = 0;
    }

    void acelerar() {
        velocidade += 10;
    }

    void frear() {
        velocidade -= 10;
    }

    String exibirVelocidade() {
        return "Velocidade atual: " + velocidade + " km/h";
    }
}

public class questao2 {
    public static void main(String[] args) {
        Carro1 carro = new Carro1("Mitsubishi", "lancer", 2012);
        carro.acelerar();
        System.out.println(carro.exibirVelocidade());
        carro.acelerar();
        System.out.println(carro.exibirVelocidade());
        carro.frear();
        System.out.println(carro.exibirVelocidade());
        carro.frear();
        System.out.println(carro.exibirVelocidade());
    }
}

File: Lista de exercicio/projetoPYTHON/questao2/questao2.py
class Carro:
    def __init__(self, marca, modelo, ano):
        self.marca = marca
        self.modelo = modelo
        self.ano = ano
        self.velocidade = 0

    def acelerar(self):
        self.velocidade += 10

    def frear(self):
        self.velocidade -= 10

    def exibir_velocidade(self):
        return f"Velocidade atual: {self.velocidade} km/h"

# Exemplo de uso:

def executar_questao2():
    carro = Carro("Mitsubishi", "lancer", 2012)
    carro.acelerar()
    print(carro.exibir_velocidade())
    carro.acelerar()
    print(carro.exibir_velocidade())
    carro.frear()
    print(carro.exibir_velocidade())
    carro.frear()
    print(carro.exibir_velocidade())
